scipy/spatial/transform/rotation.py

```diff
@@ -195,3 +196,58 @@ def from_dcm(cls, dcm):
             return cls(quat[0], normalized=True)
         else:
             return cls(quat, normalized=True)
+
+    @classmethod
+    def from_rotvec(cls, rot_vecs):
+        """Initialize class from rotation vector.
+
+        A rotation vector is a 3 dimensional vector which is co-directional to
+        the axis of rotaion and whose norm gives the angle of rotation (in
+        radians).
+
+        Parameters
+        ----------
+        rot_vecs : array_like, shape (N, 3) or (3,)
+            A single vector or a stack of vectors, where `rot_vec[i]` gives
+            the ith rotation vector.
+        """
+        is_single = False
+        rot_vecs = np.asarray(rot_vecs, dtype=float)
+
+        if rot_vecs.ndim not in [1, 2] or rot_vecs.shape[-1] != 3:
+            raise ValueError("Expected `rot_vec` to have shape (3,) "
+                             "or (N, 3), got {}".format(rot_vecs.shape))
+
+        # If a single vector is given, convert it to a 2D 1 x 3 matrix but
+        # set self._single to True so that we can return appropriate objects
+        # in the `as_...` methods
+        if rot_vecs.shape == (3,):
+            rot_vecs = rot_vecs[None, :]
+            is_single = True
+
+        num_rotations = rot_vecs.shape[0]
+
+        norms = np.linalg.norm(rot_vecs, axis=1)[:, None]
+        small_angle = norms <= 1e-3
+
+        quat = np.empty((num_rotations, 4))
+
+        zero_norms = np.nonzero(norms == 0)[0]
+        quat[zero_norms, :3] = np.array([0, 0, 0])
+
+        small_angle = np.nonzero(np.logical_and(
+            norms <= 1e-3, norms != 0))[0]
+        quat[small_angle, :3] = (0.5 - np.power(norms[small_angle], 2) /
```

Comment:
Why do you prefer to use `power` instead of `**`?

Reply:
In this particular case, I read that the results of `np.power` and `**` can sometimes differ, not enough to be seen by `np.allclose` where tolerance is `1e-8` by default. Since I'm not sure how much of a concern this is, I'll use `**`.